app/src/main/java/org/literacyapp/authentication/fallback/StudentRegistrationActivity.java

```diff
@@ -220,7 +224,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                 studentDao.insert(student);
                 Log.i(getClass().getName(), "Student stored in database with id " + student.getId());
```

Comment:
The student is updated automatically by greenDao after studentDao.insert(student) has been executed. So you could retrieve the studentId with student.getId()